app/test_main.py

```diff
@@ -1,3 +1,49 @@
+import pytest
 from app.main import get_human_age
 
-# write your code here
+
+@pytest.mark.parametrize("cat_age,dog_age,expected", [
+    (14, 14, [0, 0]),
+    (15, 15, [1, 1]),
```

Comment:
The expected result for (15, 15) is [1, 1], which assumes that the first year is fully counted as 1 human year. Ensure that this aligns with the intended logic in `convert_to_human`.